content/wiki/capitalization_scheme.md

```diff
@@ -0,0 +1,13 @@
++++
+title = "Capitalization scheme for Linebender projects"
++++
+
+All Linebender projects should follow this capitalization scheme:
+
+- The project's name should start with an uppercase letter when referring to the project itself, eg Vello.
+- The name should be all-lowercase where referring to the project as a Rust crate (for instance, in relation to crates.io). In that case, it's encouraged to use backticks to put the project name in a codeblock, eg `vello`.
+- The name should be all-lowercase where referring to the project as a Github repository. In that case, it's encouraged to either use backticks or make it a link, eg [vello#1](https://github.com/linebender/vello/pull/1).
```

Comment:
Reminder that we're aiming for one sentence per line now in the wiki

Reply:
After trying it, I'm not really a fan of this formatting style. There's no obvious way to configure it in VsCode, it looks kind of off in the editor, and I doubt the version control benefits are that big.